### ISSUE TITLE
Get proper genus label for the Separate_segmented_train_test_splits_80_20 metadata file

### ISSUE THREAD

**Issue by egrace479** [Imageomics/2018-NEON-beetles-processing#14]
[Genera list](https://huggingface.co/datasets/imageomics/2018-NEON-beetles/discussions/18#67bcabad85e7281434139a8a) for these images, and [challenge in aligning them](https://huggingface.co/datasets/imageomics/2018-NEON-beetles/discussions/18#67bd0da2eec30af7b2e0c108). I would like to add the genera to the [metadata file](https://huggingface.co/datasets/imageomics/2018-NEON-beetles/tree/main/Separate_segmented_train_test_splits_80_20/metadata.csv) to make it more usable.

Documenting this to open an issue for resolution.

_Originally posted by @egrace479 in https://github.com/Imageomics/2018-NEON-beetles-processing/issues/12#issuecomment-2680024406_

**Comment by egrace479** [comment]
Individual images are being added for _all_ the beetles in the next revision of the dataset itself, and the genera were not used in the known downstream tasks completed with these images. As a result, this alignment does not seem to be worth the effort.
